home-manager/home-manager

```diff
@@ -392,16 +392,17 @@ $xdgVars
     # '';
   };
 
-  # You can also manage environment variables but you will have to manually
-  # source
+  # Home Manager can also manage your environment variables through
+  # 'home.sessionVariables'. If you don't want to manage your shell through 
+  # Home Manager but still want it to manage env vars you'll have to manually source
```

Comment:
I think this can be simplified a bit to something like:

```suggestion
  # Home Manager then you have to manually source
```

Beside being a bit shorter, this has the benefit of making it a bit more forceful. We'll want to avoid users skipping this sourcing since quite a few modules would not work correctly if they can't set session variables.